src/components/Avatar/useIdColorHash.ts

```diff
@@ -15,11 +15,12 @@ limitations under the License.
 */
 
 /**
- * Determines a color to assign to a given username
- * @param id - a Matrix username ID
- * @returns a hexadecimal color
+ * Determines a number for a given Matrix ID, helps disambiguating users
+ * who are trying to impersonate someone else.
+ * @param id - a Matrix ID
+ * @returns a hash of the Matrix ID
  */
-export function useUsernameColor(id: string): number {
+export function useIdColorHash(id: string): number {
```

Comment:
Does it only apply to MXIDs or can it also be used for other Ids, such as room Ids? Just asking because I wonder if this and Avatar should be a bite more generic to cover rooms as well…